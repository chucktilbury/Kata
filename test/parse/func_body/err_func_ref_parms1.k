;-------------------------------------------------------------------------------
; Function reference where the output parameters are incorrect
;-------------------------------------------------------------------------------

func name1()() {
	
	name2()(123)
}
