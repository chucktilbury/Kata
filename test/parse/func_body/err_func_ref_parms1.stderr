Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_func_ref_parms1.k: 7: 13: expected a compound name or a ')' but got literal signed
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_func_ref_parms1.k: 9: 1: expected an optional list of names but got end of file
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_func_ref_parms1.k: 9: 1: expected function body element but got end of file
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_func_ref_parms1.k: 9: 1: expected function body but got end of file
